actions/tor

```diff
@@ -68,6 +69,9 @@ def parse_arguments():
 
 def subcommand_setup(_):
     """Setup Tor configuration after installing it."""
+
+    subprocess.run(['tor-instance-create', 'plinth'])
+
     aug = augeas_load()
```

Comment:
Augeas produces an error "Unable to save to file!". The reason is that the torrc file has the line "+SocksPort auto"
but our current tor augeas lens (https://github.com/freedombox/Plinth/blob/master/data/usr/share/augeas/lenses/tor.aug) does not handle lines beginning with +.

We should add this line as a test case to https://github.com/freedombox/Plinth/blob/master/data/usr/share/augeas/lenses/tests/test_tor.aug, and add + to the list of key characters (kc) in https://github.com/freedombox/Plinth/blob/master/data/usr/share/augeas/lenses/tor.aug#L25.